src/main/java/com/mercadopago/client/point/PointPaymentIntentListRequest.java

```diff
@@ -0,0 +1,27 @@
+package com.mercadopago.client.point;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Builder;
+import lombok.Getter;
+
+/** Responsible for setting date range for getting payment intent list. */
+@Getter
+@Builder
+public class PointPaymentIntentListRequest {
+  private final LocalDate startDate;
+  private final LocalDate endDate;
```

Comment:
Faltaram os comments

```suggestion
  private final LocalDate startDate;
  
  private final LocalDate endDate;
```

Reply:
Feito.